nginx.conf

```diff
@@ -52,14 +52,36 @@ http {
       # allow for HTML "meta redirects" on pages of sites that have been migrated
       # to Federalist. For context, see https://github.com/18F/federalist-proxy/issues/19.
       #
-      # More extensions can be added inside the parentheses, separated 
+      # More extensions can be added inside the parentheses, separated
       # by | characters, e.g. `\.(cfm|php)$`.
       location ~* \.(cfm)$ {
         add_header Content-Type "text/html";
         # Notice there is no trailing slash on the proxy_pass url here.
         proxy_pass <%= ENV["FEDERALIST_S3_BUCKET_URL"] %>;
       }
 
+      # Enable CORS support.
+      # Based on https://enable-cors.org/server_nginx.html
+      set $cors_origins 'https://s.codepen.io';
+      set $cors_methods 'GET, OPTIONS';
+      set $cors_headers 'DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Content-Range,Range';
+
+      if ($request_method = 'OPTIONS') {
+        add_header 'Access-Control-Allow-Origin' $cors_origins;
+        add_header 'Access-Control-Allow-Methods' $cors_methods;
+  
+        # Custom headers and headers various browsers *should* be OK with but aren't
+        add_header 'Access-Control-Allow-Headers' $cors_headers;
```

Comment:
for `OPTIONS`, https://enable-cors.org/server_nginx.html also includes

```
add_header 'Access-Control-Max-Age' 1728000;
add_header 'Content-Type' 'text/plain; charset=utf-8';
add_header 'Content-Length' 0;
return 204;
```

I left those directives out because I think the `proxy-pass` to S3 will take care of that stuff.